### PR TITLE
docs: embeddings classes

### PR DESCRIPTION
**Pull request by leo-gan**
Added a table with all Embedding classes.

### CODE REVIEW

File: docs/docs/integrations/text_embedding/index.mdx [hunk 1]
```diff
@@ -0,0 +1,114 @@
+---
+sidebar_position: 0
+sidebar_class_name: hidden
+---
+
+# Embedding
```
Comment (isahers1) [inline docs/docs/integrations/text_embedding/index.mdx:6]: Can you make title say "Embedding Models" instead?

**Comment by isahers1** [comment]
@leo-gan Is there any way to automate table generation? See docs/scripts/model_feat_table.py for reference

**Comment by leo-gan** [comment]
> @leo-gan Is there any way to automate table generation? See docs/scripts/model_feat_table.py for reference

It is automated :)

**Review by isahers1 — COMMENTED** [review]
lgtm!

**Comment by leo-gan** [comment]
@isahers1 Thanks!

**Comment by leo-gan** [comment]
> > @leo-gan Is there any way to automate table generation? See docs/scripts/model_feat_table.py for reference
> 
> It is automated :)

I've missed the link to this [automation code](https://github.com/langchain-ai/langchain/pull/22336). Maybe you can help with review.

**Comment by isahers1** [comment]
> > > @leo-gan Is there any way to automate table generation? See docs/scripts/model_feat_table.py for reference
> > 
> > 
> > It is automated :)
> 
> @isahers1 I've missed the link to this [automation code](https://github.com/langchain-ai/langchain/pull/22336). Maybe you can help with review.

Yes, will take a look tmrw!

**Pull request merged**